### ISSUE TITLE
more woes on osx

### ISSUE THREAD

**Issue by dcsan** [fourseven/meteor-scss#66]
trying to run an app on a new machine that uses meteoric and hitting problems with fourseven again

Error: not a tracked temp dir: /Users/dc/.meteor/packages/fourseven_scss/.1.0.0.k7gvb2++os.osx.x86_64+web.browser+web.cordova

```                                                                                   
/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/dev_bundle/lib/node_modules/fibers/future.js:173
						throw(ex);
						      ^
Error: not a tracked temp dir: /Users/dc/.meteor/packages/fourseven_scss/.1.0.0.k7gvb2++os.osx.x86_64+web.browser+web.cordova
    at Error (<anonymous>)
    at Object.files.freeTempDir (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/files.js:544:11)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/tropohouse.js:348:17
    at Array.forEach (native)
    at Function._.each._.forEach (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:79:11)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/tropohouse.js:347:11
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:352:18
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:345:34
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:343:23
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.enterJob (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:317:26)
    at Object.download (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/tropohouse.js:303:20)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/tropohouse.js:412:22
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:352:18
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:345:34
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:343:23
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.enterJob (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:317:26)
    at [object Object]._.extend.downloadPackagesMissingFromMap (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/tropohouse.js:409:20)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:600:25
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:352:18
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:345:34
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:343:23
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.enterJob (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:317:26)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:599:20
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:264:13
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:257:29
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:255:18
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:246:23
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.capture (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:245:19)
    at Object.catalog.runAndRetryWithRefreshIfHelpful (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/catalog.js:92:31)
    at [object Object]._.extend._downloadMissingPackages (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:598:13)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:210:35
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:352:18
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:345:34
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:343:23
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.enterJob (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:317:26)
    at [object Object]._.extend._completeStagesThrough (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:200:18)
    at [object Object]._.extend.saveChangedMetadata (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:188:10)
    at [object Object]._.extend.prepareProjectForBuild (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/project-context.js:193:10)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/commands.js:1078:20
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:264:13
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:257:29
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:255:18
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:246:23
    at [object Object]._.extend.withValue (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/fiber-helpers.js:115:14)
    at Object.capture (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/buildmessage.js:245:19)
    at Object.main.captureAndExit (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/main.js:259:29)
    at Command.main.registerCommand.name [as func] (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/commands.js:1077:8)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.yk6tjd++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/main.js:1300:23
```

**Comment by johanbrook** [comment]
I get the same when deploying to Meteor.com. Vanilla install.

**Comment by dcsan** [comment]
I ran the install again, perhaps this is related to the other error that happens with imports the first time you run (although it looks completely different)

**Comment by fourseven** [comment]
Seems to not be specific to (just) this package - https://github.com/meteor-velocity/html-reporter/issues/60 - though that doesn't help solve it. I'll open an issue with meteor/meteor

**Comment by dcsan** [comment]
i just got this err on a fresh deploy, with 2.0

deployed again and it seems ok. so it seems related to a "first run" situation

```
                                                                                   
/Users/dc/.meteor/packages/meteor-tool/.1.0.40.prjwsp++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/dev_bundle/lib/node_modules/fibers/future.js:173
						throw(ex);
						      ^
Error: not a tracked temp dir: /Users/dc/.meteor/packages/fourseven_scss/.2.0.0.14913uf++os.osx.x86_64+web.browser+web.cordova
    at Error (<anonymous>)
    at Object.files.freeTempDir (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.prjwsp++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/files.js:544:11)
    at /Users/dc/.meteor/packages/meteor-tool/.1.0.40.prjwsp++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/tropohouse.js:348:17
    at Array.forEach (native)
    at Function._.each._.forEach (/Users/dc/.meteor/packages/meteor-tool/.1.0.40.prjwsp++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:79:11)

```

**Comment by fourseven** [comment]
This has been fixed with meteor `1.0.3.2`
